content/de/wallets/Coinbase.md

```diff
@@ -0,0 +1,42 @@
++++
+type = "wallet"
+
+name = "Coinbase"
```

Comment:
This absolutely needs to be "Coinbase Wallet App" vs "Coinbase".  Coinbase as a service is custodial, and is of course easily confused with the Coinbase Wallet App which is self/non custodial.

Reply:
Are the links provided correct for the wallet?
is just the name that have to be changed?